### ISSUE TITLE
Add support for folders plugin

### ISSUE THREAD

**Issue by cniweb** [tomkuijsten/vsjenkinsmanager#34]
Add support for folders plugin (https://wiki.jenkins-ci.org/display/JENKINS/CloudBees+Folders+Plugin), please.

**Comment by tomkuijsten** [comment]
Thanks for the suggestion. Could you be a bit more specific? Is the manager not working with a Jenkins installation with the folder plugin? Or did you expect a folder structure in the "views" dropdown?

**Comment by cniweb** [comment]
Yes, i did expect a folder structure in the "views" dropdown?

**Comment by tomkuijsten** [comment]
Do you have a server where I can test the Jenkins API with this folder plugin? And how do you think the dropdown should look like with folder support?

**Comment by cniweb** [comment]
No, i hve no public Jenkins Server.
I think, as well as in Jenkins:
http://jenkins-enterprise.cloudbees.com/docs/user-guide-docs/folder-sect-usage.html

**Comment by cniweb** [comment]
Are you coming along? Can I help you?

**Comment by tomkuijsten** [comment]
Kind of busy at the moment with other things. If you want you can send me a pull request, I'll gladly accept that. Otherwise, I don't think I will implement it soon. I think that most people can work without it, and I just don't have enough spare time at the moment to pick this up, sorry.

**Comment by tomkuijsten** [comment]
Released (1.0.0.4), please test your situation and close the issue when it's fixed.

**Comment by cniweb** [comment]
In Version 1.0.0.5 the issue is not fixed.

**Comment by tomkuijsten** [comment]
I posted the previous comment at the wrong issue, my apologies, this is still not fixed. Are you busy with a pull request or not?

**Comment by cniweb** [comment]
No, I'm waiting for the implementation, because I do not know at what point I can make the change.

**Comment by tomkuijsten** [comment]
Problem is that I can't figure out how to check if a job is a folder in the json result, any ideas?
________________________________

**Comment by christ66** [comment]
@tomkuijsten it is not (easily) possible to obtain if it is a folder or freestyle project (might be able to do some tricks with `depth=1` or a depth of more than one). I created https://github.com/stapler/stapler/pull/51 which will add a `class` element which will help.

**Comment by cniweb** [comment]
Thanks @christ66 :+1:

**Comment by tomkuijsten** [comment]
That would help a lot! There is always the possibility to just "scan" every api on every job url and test if there are subprojects, but that's a very inefficient way.

**Comment by cniweb** [comment]
But is a possibility way.
@christ66 Have you another idea?

**Comment by cniweb** [comment]
@tomkuijsten @christ66 
Possibly could help this:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=421352

**Comment by cniweb** [comment]
@tomkuijsten @christ66
How does it go on here?

**Comment by tomkuijsten** [comment]
There is not much going on. I'm not using Jenkins anymore and I don't have enough time to implement this. I'll ad a help-wanted tag, but I don't expect much response. I'm gonna update the readme that I'll stop supporting this and anyone interested can claim this repo.

**Comment by cniweb** [comment]
@zachjacobs @dinowang @MartinWeijsenfeld you are interested?
